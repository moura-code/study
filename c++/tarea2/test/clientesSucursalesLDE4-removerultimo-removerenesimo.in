crearClientesSucursalesLDE
crearClientesABB
crearCliente 1 Zafiro Lopez 22
agregarAClientesABB
agregarAClientesSucursalesLDE
cantidadClientesSucursalesLDE
removerUltimoClientesSucursalesLDE
cantidadClientesSucursalesLDE
crearClientesABB
crearCliente 2 Filemon Fernandez 32
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 3 Brunilda Sanchez 41
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 4 Filemon Martinez 44
agregarAClientesABB
agregarAClientesSucursalesLDE
removerNesimoClientesSucursalesLDE 3
removerUltimoClientesSucursalesLDE
cantidadClientesSucursalesLDE
removerUltimoClientesSucursalesLDE
crearClientesABB
crearCliente 10 Valquiria Perez 33
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 11 Ubaldo Lopez 53
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 12 Rufino Rodriguez 43
agregarAClientesABB
agregarAClientesSucursalesLDE
crearClientesABB
crearCliente 13 Valquiria Rodriguez 18
agregarAClientesABB
agregarAClientesSucursalesLDE
imprimirClientesSucursalesLDE
removerNesimoClientesSucursalesLDE 3
removerNesimoClientesSucursalesLDE 1
removerNesimoClientesSucursalesLDE 2
removerNesimoClientesSucursalesLDE 1
liberarClientesSucursalesLDE
Fin
