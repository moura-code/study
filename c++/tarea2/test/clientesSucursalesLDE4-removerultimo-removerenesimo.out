Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> La SucursalesLDE de clientes ha sido creado de forma exitosa.
2> El clientesABB ha sido creado de forma exitosa.
3> Cliente creado de forma exitosa.
4> Se ha agregado el cliente al clientesABB de forma exitosa.
5> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
6> La cantidad de clientesABB en SucursalesLDE es 1
7> Ultimo clientesABB (removido):
Cliente Zafiro Lopez
Id: 1
Edad: 22
8> La cantidad de clientesABB en SucursalesLDE es 0
9> El clientesABB ha sido creado de forma exitosa.
10> Cliente creado de forma exitosa.
11> Se ha agregado el cliente al clientesABB de forma exitosa.
12> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
13> El clientesABB ha sido creado de forma exitosa.
14> Cliente creado de forma exitosa.
15> Se ha agregado el cliente al clientesABB de forma exitosa.
16> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
17> El clientesABB ha sido creado de forma exitosa.
18> Cliente creado de forma exitosa.
19> Se ha agregado el cliente al clientesABB de forma exitosa.
20> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
21> clientesABB en la posicion 3 (removido):
Cliente Filemon Martinez
Id: 4
Edad: 44
22> Ultimo clientesABB (removido):
Cliente Brunilda Sanchez
Id: 3
Edad: 41
23> La cantidad de clientesABB en SucursalesLDE es 1
24> Ultimo clientesABB (removido):
Cliente Filemon Fernandez
Id: 2
Edad: 32
25> El clientesABB ha sido creado de forma exitosa.
26> Cliente creado de forma exitosa.
27> Se ha agregado el cliente al clientesABB de forma exitosa.
28> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
29> El clientesABB ha sido creado de forma exitosa.
30> Cliente creado de forma exitosa.
31> Se ha agregado el cliente al clientesABB de forma exitosa.
32> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
33> El clientesABB ha sido creado de forma exitosa.
34> Cliente creado de forma exitosa.
35> Se ha agregado el cliente al clientesABB de forma exitosa.
36> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
37> El clientesABB ha sido creado de forma exitosa.
38> Cliente creado de forma exitosa.
39> Se ha agregado el cliente al clientesABB de forma exitosa.
40> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
41> clientesSucursalesLDE de grupos:
Grupo con edad promedio 18.00:
Cliente Valquiria Rodriguez
Id: 13
Edad: 18
Grupo con edad promedio 33.00:
Cliente Valquiria Perez
Id: 10
Edad: 33
Grupo con edad promedio 43.00:
Cliente Rufino Rodriguez
Id: 12
Edad: 43
Grupo con edad promedio 53.00:
Cliente Ubaldo Lopez
Id: 11
Edad: 53
42> clientesABB en la posicion 3 (removido):
Cliente Rufino Rodriguez
Id: 12
Edad: 43
43> clientesABB en la posicion 1 (removido):
Cliente Valquiria Rodriguez
Id: 13
Edad: 18
44> clientesABB en la posicion 2 (removido):
Cliente Ubaldo Lopez
Id: 11
Edad: 53
45> clientesABB en la posicion 1 (removido):
Cliente Valquiria Perez
Id: 10
Edad: 33
46> SucursalesLDE de ClientesABB liberada
47> Fin.
